include/grpc/grpc_security.h

```diff
@@ -530,6 +548,26 @@ GRPCAPI grpc_server_credentials* grpc_ssl_server_credentials_create(
     const char* pem_root_certs, grpc_ssl_pem_key_cert_pair* pem_key_cert_pairs,
     size_t num_key_cert_pairs, int force_client_auth, void* reserved);
 
+/** Sets the minimum TLS version that will be negotiated during the TLS
+   handshake. The caller MUST ensure that |creds| is an instance of
```

Comment:
Please align the lines (e.g. the second line needs an extra space before `handshake` and so on).

Reply:
Looking at the rest of the documentation, it looks like the extra space isn't present anywhere else either. I'm not sure why it's like this, but I think it's best to stick with the current styling.